app/models/claimToAdjustPoa/PaymentOnAccountViewModel.scala

```diff
@@ -22,15 +22,17 @@ case class PaymentOnAccountViewModel(
                              poaOneTransactionId: String,
                              poaTwoTransactionId: String,
                              taxYear: TaxYear,
-                             paymentOnAccountOne: BigDecimal,
-                             paymentOnAccountTwo: BigDecimal,
-                             poARelevantAmountOne: BigDecimal,
-                             poARelevantAmountTwo: BigDecimal,
-                             poAPartiallyPaid: Boolean
+                             totalAmountOne: BigDecimal,
+                             totalAmountTwo: BigDecimal,
+                             relevantAmountOne: BigDecimal,
+                             relevantAmountTwo: BigDecimal,
+                             poaPreviouslyAdjusted: Option[Boolean],
```

Comment:
naming convention, I guess we don't need "poa" prefix at all as are already in the context of Poa~ view model;